### PR TITLE
[Fix][Upgrade]0.7.0 upgrade 0.7.1 all bug

### PR DESCRIPTION
**Pull request by zackyoungh**
## Purpose of the pull request

0.7.0 upgrade 0.7.1 all bug

### CODE REVIEW

**Review by aiwenmo — APPROVED** [review]
LGTM

File: dlink-doc/sql/dinky.sql [hunk 1]
```diff
@@ -21,941 +21,1020 @@ SET NAMES utf8mb4;
...
-      UNIQUE KEY `un_idx1` (`name`,`tenant_id`) USING BTREE
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 ROW_FORMAT=DYNAMIC COMMENT='fragment management';
-
-
--- ----------------------------------------------------------------------------- Metadata related data table the start -------------------------------------------------------------------------------------------
+-- Table structure for dinky_alert_group
+-- ----------------------------
+DROP TABLE IF EXISTS `dinky_alert_group`;
+CREATE TABLE `dinky_alert_group`  (
+                                      `id` int(11) NOT NULL AUTO_INCREMENT COMMENT 'id',
```
Comment (aiwenmo) [inline dlink-doc/sql/dinky.sql]: Please check sql.

**Review by aiwenmo — CHANGES_REQUESTED** [review]
Check sql.

**Review by Zzm0809 — COMMENTED** [review]
LGTM

**Pull request closed without merging**